terra_geocrud/signals.py

```diff
@@ -6,23 +6,31 @@
 from geostore.helpers import execute_async_func
 from geostore.models import Feature, LayerRelation
 from geostore.signals import save_feature, save_layer_relation
-from terra_geocrud.tasks import (feature_update_relations_destinations, layer_relations_set_destinations,
-                                 feature_update_relations_origins)
+from terra_geocrud.tasks import (feature_update_relations_and_properties, layer_relations_set_destinations,
+                                 feature_update_relations_origins, feature_update_destination_properties)
 
 
 signals.post_save.disconnect(save_feature, sender=Feature)
 
 signals.post_save.disconnect(save_layer_relation, sender=Feature)
 
 
+def execute_async_save(update_fields, instance, kwargs):
+    if update_fields is None or 'geom' in update_fields:
+        execute_async_func(feature_update_relations_and_properties, (instance.pk, kwargs))
+    elif "geom" not in update_fields and "properties" in update_fields:
+        execute_async_func(feature_update_destination_properties, (instance.pk, kwargs))
```

Comment:
Est-ce que je peux te suggérer de re-écrire tes conditions pour que ce soit plus clair les cas traités/non-traités par ce signaux ?

Par exemple :
```suggestion
    if update_fields :
        if "geom" not in update_fields and "properties" in update_fields :
           # partial save => properties updated
        elif "geom" in update_fields and "properties"  not in update_fields :
           # partial save => geom updated
        elif "geom" in update_fields and "properties" in update_fields :
           # partial save => properties & geom updated
        else :
           # ???
    else :
          # full update
```

Reply:
I change for a if, else thanks to the bulk_update